### DISCUSSION TITLE
Add Requirement

### DISCUSSION THREAD

**Discussion by lutfifirdaus** [konnco/filament-import#51] [Q&A] [answered]
Hello.

I have some request for documentation because i wanna try to install this package to my project but its return some requirement problem on composer. Can we have any requirement of this package on the documentation so we know is it compatible with someone's project?

Thanks

**Comment by frankyso** [comment, marked as answer]
maybe because of laravel excel, maybe you can attach what error you get?

**Reply by lutfifirdaus** [reply]
I got this error

`require illuminate/contracts ^9.0 -> found illuminate/contracts[v9.0.0-beta.1, ..., 9.x-dev] but these were not loaded, likely because it conflicts with another require.`

I've been try to look what package that use this to try solve the problem and my conclussion is maybe its because laravel version...

**Reply by frankyso** [reply]
yes, it only support Laravel 9

**Reply by lutfifirdaus** [reply]
So could we can have requirement description on this documentation's package pls?

**Reply by frankyso** [reply]
sure, I will do that :)

or you can help me by creating a fork and updating the documentation for that :)

**Reply by lutfifirdaus** [reply]
Sure, I'll try it

**Reply by frankyso** [reply]
thank you
